doc/espnet2_tutorial.md

```diff
@@ -340,3 +340,197 @@ To enable online decoding, the argument `--use_streaming true` should be added t
+
+**Example 1: conv 2D + 2x Conv 1D + 14x Conformer.**
+
+```yaml
+encoder_conf:
+    main_conf:
+      pos_wise_layer_type: linear
+      pos_wise_act_type: swish
+      pos_enc_layer_type: rel_pos
+      conv_mod_act_type: swish
+    input_conf:
+      block_type: conv2d
+      dim_conv: 256
+      subsampling_factor: 4
+      dropout_rate_pos_enc: 0.1
+    body_conf:
+    - block_type: conv1d
+      dim_output: 128
+      kernel_size: 3
+    - block_type: conv1d
+      dim_output: 256
+      kernel_size: 2
+    - block_type: conformer
+      dim_linear: 1024
+      dim_hidden: 256
+      heads: 8
+      dropout_rate: 0.1
+      dropout_rate_pos_wise: 0.1
+      dropout_rate_att: 0.1
+      macaron_style: true
+      conv_mod_kernel: 31
+      num_blocks: 14
+```
+
+**Example 2: VGG + 4x RNN w/ projection layers.**
+
+```yaml
+encoder_conf:
+    input_conf:
+      block_type: vgg
+    body_conf:
+    - block_type: rnn
+      num_blocks: 4
+      dim_hidden: 512
+      dim_proj: 256
+```
+
+#### Decoder
+
+For the decoder, two types of blocks are available: RNN and stateless (only embedding). It is defined through two parameters: `decoder` and `decoder_conf`. The first one take a string defining the type of block (either `rnn` or `stateless`) to use while the second takes a single configuration. The following parameters can be set but are all optional:
+
+    decoder_conf:
+      rnn_type (RNN only): Type of RNN cells (int, default = "lstm").
+      dim_hidden (RNN only): Dimension of the hidden layers (int, default = 256).
+      dim_embedding: Dimension of the embedding layer (int, default = 256).
+      dropout: Dropout rate for the RNN output nodes (float, default = 0.0).
+      dropout_embed: Dropout rate for the embedding layer (float, default = 0.0).
+
+#### Joint network
+
+Currently, we only propose the standard joint network module composed of two three linear layers and an activation functions. The module definition is optional but the following parameters can be modified through the configuration parameter `joint_network_conf`:
+
+    joint_network_conf:
+      dim_joint_space: Dimension of the joint space (int, default = 256).
+      joint_act_type: Type of activation in the joint network (str, default = "tanh").
+
+### Multi-task learning
+
+We also support multi-task learning with two auxiliary tasks: CTC and cross-entropy w/ label smoothing option (called LM loss here). The auxiliary tasks contributes to the overal task defined as:
+
+**L_tot = (λ_trans x L_trans) + (λ_auxCTC x L_auxCTC) + (λ_auxLM x L_auxLM)**
+
+where the losses (L_*) are respectively, in order: The Transducer loss, the CTC loss and the LM loss. Lambda values define their respective contribution to the total loss. Each task can be parametrized using the following options, passed to `model_conf`:
+
+    model_conf:
+      transducer-loss-weight: Weight of the Transducer loss (float, default = 1.0)
```

Comment:
typo: transducer_weight
suggestion: add the yaml config you are using to the librispeech conf folder and reference it ?

Reply:
> typo: transducer-loss-weight -> transducer_weight

Thanks.

> suggestion: add the yaml config you are using to the librispeech conf folder and reference it ?

It's the plan but it will be updated in another PR. Libri work won't be included here for clarity.